benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -260,6 +265,22 @@ void GraphicsBenchmarkApp::SetupSphereResources()
         samplerCreateInfo.maxLod                  = FLT_MAX;
         PPX_CHECKED_CALL(GetDevice()->CreateSampler(&samplerCreateInfo, &mMetalRoughnessTexture.sampler));
     }
+    {
+        // 1x1 White Texture
+        grfx_util::ImageOptions options = grfx_util::ImageOptions().MipLevelCount(0);
```

Comment:
I think you can also use `PPX_CHECKED_CALL(grfx_util::CreateTexture1x1<uint8_t>(GetDevice()->GetGraphicsQueue(), {255, 255, 255, 255}, &m1x1WhiteTexture));`
Example in 16_gbuffer 
https://github.com/search?q=repo%3Agoogle%2Fbigwheels%20m1x1WhiteTexture&type=code

Reply:
Thanks, I didn't know this existed. Changed